charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -49,6 +49,12 @@ data:
     memory.heap-headroom-per-node={{ .Values.server.config.memory.heapHeadroomPerNode }}
     discovery-server.enabled=true
     discovery.uri=http://localhost:{{ .Values.service.port }}
+{{- if .Values.server.config.processForwarded }}
+    http-server.process-forwarded={{ .Values.server.config.processForwarded }}
+{{- end }}
```

Comment:
This doesn't seem useful to me, having a separate value to just set a single config in a single file doesn't feel useful. Mostly because going that route means we'd have a lot of values in the chart - one for almost each config property which can already be specified if needed.

Reply:
Contrast this with password auth values - it is useful since it sets proper values across multiple files and places.